src/parser.rs

```diff
@@ -4553,7 +4553,10 @@ impl<'a> Parser<'a> {
                 charset_name,
                 collation_name,
             })
-        } else if self.consume_token(&Token::Eq) || self.parse_keyword(Keyword::TO) {
+        } else if (self.consume_token(&Token::Eq) || self.parse_keyword(Keyword::TO))
+            || (variable == ObjectName(vec!["TIMEZONE".into()]))
+        {
+            // when the object name is TIMEZONE, we support `SET TIMEZONE 'UTC'` without Eq sign or TO
```

Comment:
@alamb @435201823

`SET TIMEZONE 'UTC'` is an exceptional case in posetgresql 
not sure whether it's a good idea to add it here